### PR TITLE
Added calls for project phases and changed README.

### PR DESCRIPTION
**Pull request by jrutschke**
Added calls to create, update, get and delete phases.
Testcases were added and README updated.  New phase testcases passed.

This time with changes clearly visible.

### CODE REVIEW

File: tests/test_float_api.py [hunk 1]
```diff
@@ -689,3 +708,54 @@ def test_create_get_delete():
...
+
+ # Update a phase
+  name = random_string(32)
+  phase = api.update_phase(
+    phase_id = phase['phase_id'],
+    name = name
+    )
+  assert isinstance(phase, dict), "Updated phase is a dict"
+  assert set(phase_keys()).issubset(phase.keys()), "Updated client has all keys"
+  assert phase['name'] == name, "Name of client is updated"
```
Comment (tobiasbp) [inline tests/test_float_api.py:749]: Name of phase, nor client.

File: tests/test_float_api.py [hunk 2]
```diff
@@ -689,3 +708,54 @@ def test_create_get_delete():
...
 
+# Create, update and delete a phase
+def test_phase():
+
+  # Create a test project
+  project = api.create_project(name=random_string(32))
+  assert isinstance(project, dict), "New project is a dict"
+  assert set(project_keys()).issubset(project.keys()), "New project has all keys"
+
+  # Create a test person
```
Comment (tobiasbp) [inline tests/test_float_api.py:719]: Test person does not seem to be needed when testing phase?

File: tests/test_float_api.py [hunk 3]
```diff
@@ -689,3 +708,54 @@ def test_create_get_delete():
...
+  r = api.delete_phase(phase['phase_id'])
+  assert r == True
+
+  # Delete test person
+  r = api.delete_person(person['people_id'])
+  assert r == True
+
+  # Delete test project
+  r = api.delete_project(project['project_id'])
+  assert r == True
```
Comment (tobiasbp) [inline tests/test_float_api.py:761]: Please add a new line.

File: tests/test_float_api.py (hunk 1 above)
Comment (tobiasbp) [inline tests/test_float_api.py:749]: Name of phase, nor client.

**Review by tobiasbp — CHANGES_REQUESTED** [review]
Please look at my comments and resolve issues.

**Comment by jrutschke** [comment]
Thanks, changed.

**Review by tobiasbp — APPROVED** [review]
Approved. Thanks for the contribution. I need to release an updated package on PyPI.

**Pull request merged**